examples/plugin/Hello_world/Hello_world.cc

```diff
@@ -0,0 +1,40 @@
+#include <ignition/gazebo/Model.hh>
+#include <ignition/gazebo/System.hh>
+#include <ignition/gazebo/Util.hh>
+
+
+namespace ignition;
+namespace gazebo;
+namespace systems;
+
+// Inherit from System and 2 extra interfaces
+// ISystemConfigure and ISystemPostUpdate
+
+class ignition::gazebo::systems::Hello_world
```

Comment:
According to our [style guide](https://ignitionrobotics.org/docs/all/contributing#style-guides), all classes must be `CamelCases`, i.e.:

```suggestion
class ignition::gazebo::systems::HelloWorld
```